src/containers/pages/FocusInvestigation/map/active/index.tsx

```diff
@@ -197,6 +200,9 @@ class SingleActiveFIMap extends React.Component<
 
     return (
       <div>
+        <Helmet>
+          <title>{FI_SINGLE_MAP_TITLE}</title>
```

Comment:
This title should include the title of the plan, something like: `Focus Investigation: A1 - Tha Sen 8- Focus 01 |  Reveal`